router/routes.php

```diff
@@ -10,15 +10,15 @@
 /**
  * Setup routes with a single request method:
  *
- * $app->get('/', App\Handler\HomePageHandler::class, 'home');
- * $app->post('/album', App\Handler\AlbumCreateHandler::class, 'album.create');
- * $app->put('/album/:id', App\Handler\AlbumUpdateHandler::class, 'album.put');
- * $app->patch('/album/:id', App\Handler\AlbumUpdateHandler::class, 'album.patch');
- * $app->delete('/album/:id', App\Handler\AlbumDeleteHandler::class, 'album.delete');
+ * $app->get('/', [App\Handler\HomePageHandler::class], 'home');
+ * $app->post('/album', [App\Handler\AlbumCreateHandler::class], 'album.create');
+ * $app->put('/album/:id', [App\Handler\AlbumUpdateHandler::class], 'album.put');
+ * $app->patch('/album/:id', [App\Handler\AlbumUpdateHandler::class], 'album.patch');
+ * $app->delete('/album/:id', [App\Handler\AlbumDeleteHandler::class], 'album.delete');
  *
  * Or with multiple request methods:
  *
- * $app->route('/contact', App\Handler\ContactHandler::class, ['GET', 'POST', ...], 'contact');
+ * $app->route('/contact', [App\Handler\ContactHandler::class], ['GET', 'POST', ...], 'contact');
```

Comment:
@peter279k My apologies, if you want I can make myself, I want to mean to delete this last comment block as in https://github.com/antidot-framework/reactive-antidot-starter/pull/12/files

Reply:
Thanks. Deleted.